react/features/overlay/components/OverlayContainer.js

```diff
@@ -95,6 +121,25 @@ class OverlayContainer extends Component {
     }
 
     /**
+     * Initializes a new ReloadTimer instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     * @public
+     */
+    constructor(props) {
+        super(props);
+        this.state = {
+            /**
+             * Indicates whether the film strip only mode is enabled or not.
+             *
+             * @type {boolean}
+             */
+            filmStripOnly: interfaceConfig.filmStripOnly
```

Comment:
Yes it will be. But at least in my head the state is closer to props than anything else. So I think it make sense to leave it as it is. Although I don't have strong opinion on this one because I also understand the argument that since it is not changing now and this implementation is not handling changes it can be outside of the state. @lyubomir do you have any opinion on this?

Also